index.src.html

```diff
@@ -650,6 +655,17 @@ <h4 id="html-iframe-element-attributes">Attributes</h4>
   The <code>referrerPolicy</code> IDL attribute must <a>reflect</a> the
   <code>referrerpolicy</code> content attribute, <a>limited to only known
   values</a>.
+
+  <h3 id="html-link-element">HTMLLinkElement</h3>
+  <pre class="idl">
+    partial interface HTMLLinkElement {
+      attribute DOMString referrerPolicy;
+    };
+  </pre>
+  <h4 id="html-link-element-attributes">Attributes</h4>
+  The <code>referrerPolicy</code> IDL attribute must <a>reflect</a> the
+  <code>referrerpolicy</code> content attribute, <a>limited to only known
+  values</a>.
```

Comment:
Sorry for the delay. I still don't really get what's special about the link element that it needs special treatment here. The integration with Fetch is already defined by using Section 8.3 "Determine request's Referrer", which includes "If request was initiated by an element with a referrerpolicy content attribute..."